### PR TITLE
Final Update HomePage and Navbar (sans Search, cart, and footer)

### PR DESCRIPTION
**Pull request by arecesj**
- Renders the Homepage without search and cart functionality. No footer (that's next).
- Built Homepage navbar and products components. All are connected dynamically via Contentful.
- Connected project to contentful. Added info to netlify (preview available there)
- link: https://deploy-preview-15--determined-benz-390e9b.netlify.com/

### CODE REVIEW

File: src/components/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import NavButtons from './molecules/NavButtons'
+import { StaticQuery, graphql } from 'gatsby'
+
+/** Overview of NavBar component:
+ *  - Created a styled div NavContainer which wraps our NavBar
+ *  - NavBar renders NavButtons and Banner Promo
+ *  - NavButtons and Banner Promo will be passed down props such as icons and text that are queried dynamically from contentful
+ *
+ */
+//Another comment
```
Comment (alissarenz) [inline src/components/NavBar.jsx]: ??

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { StaticQuery, graphql } from 'gatsby'
  *  - NavButtons and Banner Promo will be passed down props such as icons and text that are queried dynamically from contentful
  *
  */
-
+//Another comment
```
Comment (alissarenz) [inline src/components/Navbar.jsx]: ??

File: src/components/NavBar.jsx (hunk 1 above)
Comment (arecesj) [reply src/components/NavBar.jsx]: We had that random comment when we were trying to change the name of the file but git config was ignoring it.

Comment (arecesj) [reply src/components/NavBar.jsx]: We'll remove it

File: src/components/Navbar.jsx (hunk 2 above)
Comment (arecesj) [reply src/components/Navbar.jsx]: ibid

**Pull request merged**